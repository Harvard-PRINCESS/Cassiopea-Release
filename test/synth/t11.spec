(* this is the same as verify/t02.spec *)
let x0: 8 bit = *r0
let x1: 8 bit = *r1
let x2: 8 bit = *r2
let x3: 8 bit = *r3
frame:
pre: true
post: *r0 == x0 b+ x1 b+ x2 b+ x3
