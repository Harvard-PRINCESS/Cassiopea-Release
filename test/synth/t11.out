(ADD r0 r3)
(ADD r0 r1)
(ADD r0 r2)
